.travis.yml

```diff
@@ -1,9 +1,23 @@
 language: vim
+rvm:
+  - 1.8.7 # Use stock
+  - 1.9.2 # Build latest
```

Comment:
@cirosantilli Test execution isolation. If one version of vim fails (say 7.4.x cuz of new/changed behaviour) it shouldn't crash the stock build. If you ran them one after the other, on any execution you'd have to check the log & see which of the two sets of vader tests failed.